pep-9999.rst

```diff
@@ -585,6 +585,25 @@ A good stub definition::
 Finally, even in the presence of ``__getattr__`` and ``__setattr__``, it is
 still recommended to separately define known attributes.
 
+Constants
+---------
+
+Type constants using ``Literal``, instead of using a generic type.
```

Comment:
"generic type" is a little confusing here - I first thought of typing.Generic and parameterizable types. How about something along the lines of:

When the value of a constant is important, annotate it using ``Literal`` instead of its type.